tools/delete_pubspec_overrides.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:io';
```

Comment:
I guess we have 0 package dependencies in this script, so no pubspec is needed? it feels a bit weird though.

Does it mean this script always has to be the latest language version for example?

Reply:
I never use pubspecs on scripts :D It just means no restrictions on the SDK used, right?

Reply:
Which could mean it stops working on Dart 4.0 ?

Fine with me either way. 4.0 is far out =P